packages/commonwealth/client/scripts/views/pages/overview/index.tsx

```diff
@@ -13,29 +15,28 @@ import { CWIconButton } from '../../components/component_kit/cw_icon_button';
 import { CWText } from '../../components/component_kit/cw_text';
 import { PageLoading } from '../loading';
 import { TopicSummaryRow } from './topic_summary_row';
-import useUserActiveAccount from 'hooks/useUserActiveAccount';
 
 const OverviewPage = () => {
   const navigate = useCommonNavigate();
   const forceRerender = useForceRerender();
   const { isWindowExtraSmall } = useBrowserWindow({});
   const { activeAccount: hasJoinedCommunity } = useUserActiveAccount();
 
+  const { data: recentlyActiveThreads } = useFetchThreadsQuery({
+    queryType: 'active',
+    chainId: app.activeChainId(),
+    topicsPerThread: 3
+    // TODO: ask for a pinned thread prop here to show pinned threads
```

Comment:
This is calling the `/threads?active=true` API, can we add a query param to return pinned threads when applicable? @rbennettcw

Reply:
Sure, please create a ticket for it and assign to me

Reply:
Ticket here https://github.com/hicommonwealth/commonwealth/issues/4671

cc @jnaviask @rbennettcw